pkgs/top-level/all-packages.nix

```diff
@@ -6291,10 +6291,10 @@ with pkgs;
     lfe lfe_1_2
     erlangR16 erlangR16_odbc
     erlang_basho_R16B02 erlang_basho_R16B02_odbc
-    erlangR17 erlangR17_odbc erlangR17_javac erlangR17_odbc_javac
-    erlangR18 erlangR18_odbc erlangR18_javac erlangR18_odbc_javac
-    erlangR19 erlangR19_odbc erlangR19_javac erlangR19_odbc_javac
-    erlangR20 erlangR20_odbc erlangR20_javac erlangR20_odbc_javac;
+    erlangR17 erlangR17_odbc erlangR17_javac erlangR17_odbc_javac erlangR17_nox
```

Comment:
@LnL7 These lines expose plenty of Erlang versions at top level in all-packages.nix.

Reply:
Indeed, but I made a pr to remove them a while back https://github.com/NixOS/nixpkgs/commit/0fafa0d7d7c89dfb8c3c7fbc5977cfe7959ebb78